README.md

```diff
@@ -0,0 +1,23 @@
+## Contents Of This File
+
+ * About EzContent_api
+ * Key features
+ * Drupal 8 Installation
+
+## About EzContent_api
+
+This module adds JSON API functionality to EZContent
+(https://www.drupal.org/project/ezcontent) profile. This module extends the 
+JSON API module provided by Drupal core to enabled the following features:
+
+## Key Features
+
+-> Preview unpublished content in de-coupled front-ends.
+-> Use Drupal's layout builder to create landing pages with de-coupled 
+front-ends.
+-> Starter kit for using de-coupled front ends with EzContent API.
+https://github.com/srijanone/ezcontent-decoupled/blob/master/README.md
+
+## Drupal 8 Installation Via Drupal Composer
+
+composer require 'drupal/ezcontent_api:^1.0'
```

Comment:
Make use of code block here, please see https://github.com/srijanone/ezcontent/blob/8.x/README.md for reference.